src/debugpy/common/util.py

```diff
@@ -148,3 +148,18 @@ def srcnameof(obj):
         name += ")"
 
     return name
+
+
+def skip_trace():
+    """Returns True if tracing should be skipped for the current thread."""
+    return "DEBUGPY_TRACE_DEBUGPY" not in os.environ
+
+
+def disable_tracing(thread):
```

Comment:
Let's call this something like `hide_thread_from_debugger` to distinguish it from other uses of "tracing" (that pertain to the debuggee) elsewhere in the code. 

Similarly for `skip_trace` - something like `hide_debugpy_internals` would be clearer, and will cover any other future tweaks we need to do to keep this working.

Reply:
Sounds good, will do.